README.md

```diff
@@ -31,6 +31,27 @@ end
 
 NOTE: If you don't already have an environment variable for your Mandrill API key, don't forget to create one.
 
+### Mandrill Templates
+
+If you want to use this gem with mandrill templates you just have to add the `template` param to the `mail` function.
+
+```ruby
+class MyMailer < ActionMailer::Base
+  def notify_user(email)
+    mail(
+      to: email,
+      from: 'your@email.com',
+      template: 'your-mandrill-template-slug',
+      template_content: { # optional
```

Comment:
@genaromadrid everyone will look into Mandrill's API docs and expect their syntax to work. There's absolutely no need for one more non-standard syntax, this gem should be as thin and as transparent as possible.

Please check the second PR for the same feature, I've proposed a clear way to do this. https://github.com/spovich/mandrill_dm/pull/30#issuecomment-174342931